.github/workflows/ci.yaml

```diff
@@ -0,0 +1,53 @@
+name: Tests
+on:
+  workflow_call:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
```

Comment:
Whether we'll agree to use 20.04 or 22.04 is open to discussion, but we may want to specify `ubuntu-2*.04` here, in case `ubuntu-latest` is suddenly [promoted](https://github.com/actions/runner-images) to 22.04.